ml/ml_table.yaml

```diff
@@ -28,24 +36,39 @@ datasets:
   attributes:
   - name: n_splits
     dtype: int
-    doc: the number of cross-validation splits used
+    doc: The number of cross-validation splits used
 
 - data_type_def: ClassProbability
   data_type_inc: SupervisedOutput
   dtype: float
   doc: A column for storing the class probability for each class across the samples
+  attributes:
+  - name: training_labels
```

Comment:
Is this necessary since SupervisedOutput already has this attribute?

Reply:
Good point. I missed that